src/DIRAC/Core/Tornado/Server/TornadoServer.py

```diff
@@ -209,10 +208,10 @@ def startTornado(self):
                 self.__reportToMonitoring(self.__elapsedTime), self.__monitoringLoopDelay * 1000
             ).start()
 
-            # If we are running with python3, Tornado will use asyncio,
-            # and we have to convince it to let us run in a different thread
-            # Doing this ensures a consistent behavior between py2 and py3
-            asyncio.set_event_loop_policy(tornado.platform.asyncio.AnyThreadEventLoopPolicy())
+        # If we are running with python3, Tornado will use asyncio,
+        # and we have to convince it to let us run in a different thread
+        # Doing this ensures a consistent behavior between py2 and py3
+        asyncio.set_event_loop_policy(tornado_asyncio.AnyThreadEventLoopPolicy())
```

Comment:
I'm not sure it's worth adding an import to save a handful of characters:
```suggestion
        asyncio.set_event_loop_policy(tornado.platform.asyncio.AnyThreadEventLoopPolicy())
```

Reply:
I got following exception without this importing, something like:
```
AttributeError: module 'tornado' has no attribute 'platform'
```